MyDataHelpsKit/Model/SurveyPresentation.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  SurveyPresentation.swift
+//  MyDataHelpsKit
+//
+//  Created by CareEvolution on 3/8/23.
+//
+
+import Foundation
+import RegexBuilder
```

Comment:
I didn't see any regex parsing. Is this still needed?

Reply:
Good catch